### PR TITLE
Created Air Quality page

### LINKED ISSUE #43: Air Quality System
- src/components/projects/ AirQuality.jsx
image

### PR DESCRIPTION
**Pull request by AgentEinstein**
image

### CODE REVIEW

**Comment by AgentEinstein** [comment]
image
Latest image

File: src/components/projects/ AirQuality.jsx [hunk 1]
```diff
@@ -1,5 +1,83 @@
...
+          industrialization. This has in turn put the health of population in
+          risk of respiratory disorders, cardiovascular diseases and
+          asthma/allergy. This project focuses on creating a micro controlled
+          gas sensor system aimed at alerting the user when dangerous levels of
+          pollutants are in the air. We are also developing an automated air
+          filtration system to decrease exposure from harmful particulates in
+          the air.
+        </p>
+      </div>
+      <div>
```
Comment (menthy-wu) [inline src/components/projects/ AirQuality.jsx:19]: you probably don't need this parent div

File: src/components/projects/ AirQuality.jsx [hunk 2]
```diff
@@ -1,5 +1,83 @@
...
+          <li className="mb-2">
+            Make an impact on someone&apos;s life by improving quality of life
+            through smart sensors and ventilation technology
+          </li>
+          <li className="mb-2">
+            Gain on hands experience on how to calibrate and program sensors
+          </li>
+        </ul>
+      </div>
+      <div>
```
Comment (menthy-wu) [inline src/components/projects/ AirQuality.jsx:46]: you probably don't need this parent div

File: src/components/projects/ AirQuality.jsx [hunk 3]
```diff
@@ -1,5 +1,83 @@
...
+            Ventilation system will be controlled by gas sensing micro
+            controller to maximize power and filter life.
+          </li>
+          <li className="mb-2">
+            Gas Sensing unit will be able to alert user and recommend proper
+            procedure to avoid unnecessary exposure to harmful particulates
+          </li>
+        </ul>
+      </div>
+      <div>
```
Comment (menthy-wu) [inline src/components/projects/ AirQuality.jsx:73]: you probably don't need this parent div

**Review by menthy-wu — CHANGES_REQUESTED** [review]

File: src/components/projects/ AirQuality.jsx (hunk 1 above)
Comment (AgentEinstein) [reply src/components/projects/ AirQuality.jsx:19]: The images don't appear in the right spots unless I make a separate div for them. Do you have some method in mind I could use to implement them correctly without making a separate div for them?

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**